hpi-schul-cloud/schulcloud-client
# BC-5041 - Remove consent notice in manual registration (SC BB & NBC)

# Description

QA note:
Change was tested on env:
https://brb-bc-5041-remove-consent-notice.cd.dbildungscloud.dev/
https://brb-bc-5041-remove-consent-notice.cd.dbildungscloud.dev/
https://brb-bc-5041-remove-consent-notice.cd.dbildungscloud.dev/
No iseuss found. All acceptance criteria done.



## Links to Tickets or other pull requests
https://ticketsystem.dbildungscloud.de/browse/BC-5041


## Changes


## Data Security


## Deployment


## New Repos, NPM packages or vendor scripts


## Screenshots of UI changes


## Approval for review


> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.
